src/bundle/Resources/views/admin/content_type/modal_add_translation.html.twig

```diff
@@ -0,0 +1,3 @@
+{% include '@ezdesign/content/modal_add_translation.html.twig' with {
```

Comment:
Seems we have name collision here. ezplatform-ee-demo enables `EzPlatformAutomatedTranslationBundle` which overrides this template. Maybe this is one of the cases where we should go back to traditional Symfony/Twig way of including templates, so `@EzPlatformAdminUi/content/modal_add_translation.html.twig`.
This won't disable advantages of eZ Design. Developer will just need to explicitly override `@ezdesign/admin/content_type/modal_add_translation.html.twig`, which makes perfect sense TBH.